pom.xml

```diff
@@ -5,7 +5,7 @@
     <artifactId>onesignal-java-client</artifactId>
     <packaging>jar</packaging>
     <name>onesignal-java-client</name>
-    <version>1.2.2</version>
+    <version>1.3.0</version>
```

Comment:
This seems to indicate the version end users will use is 1.3.0 rather than 2.0.2?

Reply:
Oh you're right - it should be `2.0.2`. But it looks like we've always used the API version here.